aiosmtpd/tests/test_handlers.py

```diff
@@ -63,16 +66,23 @@ class DataHandler:
     content: AnyStr = None
     original_content: bytes = None
 
-    async def handle_DATA(self, server, session, envelope):
+    async def handle_DATA(
+        self, server: Server, session: ServerSession, envelope: Envelope
+    ) -> str:
         self.content = envelope.content
         self.original_content = envelope.original_content
         return S.S250_OK.to_str()
 
 
+class MessageHandler(AbstractMessageHandler):
+    def handle_message(self, message: Em_Message) -> None:
+        pass
```

Comment:
because `aiosmtpd.handler.Message` has been marked as an ABC, we can't use it directly without raising some qa errors. This is a simple concretion of the ABC for use in this test module.